integration-tests/src/test/java/com/linkedin/kafka/clients/consumer/LiKafkaInstrumentedConsumerIntegrationTest.java

```diff
@@ -76,7 +77,9 @@ public void testConsumerLiveConfigReload() throws Exception {
     producer.flush();
     producer.close(1, TimeUnit.MINUTES);
 
-    EmbeddableMario mario = new EmbeddableMario(null);
+    MarioConfiguration marioConfiguration = MarioConfiguration.embeddableInMem();
+    marioConfiguration.setEnableNgSupport(true);
```

Comment:
question: you exclude northguard from build.gradle, so why set to true here?

Reply:
fixed.